avalon/tools/cbsceneinventory/model.py

```diff
@@ -110,9 +110,7 @@ def add_items(self, items):
             representation = io.find_one({
                 "_id": io.ObjectId(representation_id)
             })
-            version = io.find_one({"_id": representation["parent"]})
-            subset = io.find_one({"_id": version["parent"]})
-            asset = io.find_one({"_id": subset["parent"]})
+            version, subset, asset, _ = io.parenthood(representation)
```

Comment:
Note that the previous was faster, and was chosen as optimization because these queries are made many many times in this interface when there is a lot of loaded content in a scene.

Reply:
I will revert this one